server/api/admin.go

```diff
@@ -61,7 +61,7 @@ func (h *adminHandler) DeleteRegionCache(w http.ResponseWriter, r *http.Request)
 		return
 	}
 	rc.DropCacheRegion(regionID)
-	if h.svr.IsAPIServiceMode() {
+	if h.svr.IsServiceIndependent(utils.SchedulingServiceName) {
```

Comment:
we also need to add judge in `pkg/utils/apiutil/serverapi/middleware.go` with ` h.s.IsServiceIndependent(rule.targetServiceName) `
![image](https://github.com/tikv/pd/assets/19542290/19baf4dd-af92-4c5a-a2d3-728786cbcdde)

Reply:
I have added it in #7297